AWS/aws-services.json

```diff
@@ -2279,5 +2279,32 @@
       "me-south-1",
       "us-east-2"
     ]
+  },
+  {
+    "service": "Macie",
```

Comment:
❓ do you need to add one for organization account and inspectorv2 account?

Reply:
Added configs for inspector and organization account